x-pack/filebeat/input/cometd/input_test.go

```diff
@@ -393,18 +393,16 @@ func assertEventMatches(t *testing.T, expected bay.MaybeMsg, got beat.Event) {
 func TestMultiEventForEOFRetryHandlerInput(t *testing.T) {
 	var err error
 
-	errorAfterEvent := 2
-	expectedHTTPEventCount := 6
-	expectedEventCount := 4
+	expectedEventCount := 2
```

Comment:
Since this is a unit test having 2 events should be fine. 
But, I hope this would work with more number of events as well.

Reply:
Yes, Input works properly with more number of events. We have kept it to 2 for unit testing purpose only.